src/golang/lib/engine/aq_engine.go

```diff
@@ -659,7 +656,7 @@ func (eng *aqEngine) TriggerWorkflow(
 	workflowId uuid.UUID,
 	name string,
 	timeConfig *AqueductTimeConfig,
-	parameters map[string]param.Param,
+	parameters map[string]string,
```

Comment:
I don't think this will compile

Reply:
oh yes I think I checked out the wrong branch's file here